src/hooks/useMaker.js

```diff
@@ -83,11 +83,19 @@ function useMaker() {
     return connectedAddress;
   };
 
+  const connectMobileProvider = async (type, callback) => {
```

Comment:
what's the advantage of using a callback instead of returning a promise as in `connectBrowserProvider`? (also note that we don't necessarily have to make this fnc async in the current case)

Reply:
also, wdyt about making this a more generic `connectToProvider` function?

Reply:
I think we can probably do without the callback since we don't really need to take action on the list of accounts before adding it into maker. I wanted to make sure we were sending the account to mixpanel, but I'll look into how that's being done with the `browserProvider`. Also, I like `connectToProviderOfType`, I'll genericize that.